hazelcast/test/src/HazelcastTests2.cpp

```diff
@@ -172,7 +172,7 @@ namespace hazelcast {
                 std::string timeString = hazelcast::util::StringUtil::time_to_string(
                         std::chrono::steady_clock::now());
                 //expected format is "%Y-%m-%d %H:%M:%S.%f" it will be something like 2018-03-20 15:36:07.280
-                ASSERT_EQ((size_t) 23, timeString.length());
+                ASSERT_LE((size_t) 21, timeString.length());
```

Comment:
```suggestion
                ASSERT_EQ(static_cast<std::size_t>(23), timeString.length());
```
setw will now ensure that the length is always 23.